CMakeLists.txt

```diff
@@ -429,27 +431,34 @@ if(WIN32)
         "${OpenMW_SOURCE_DIR}/Docs/license/DejaVu Font License.txt"
         "${OpenMW_BINARY_DIR}/settings-default.cfg"
         "${OpenMW_BINARY_DIR}/gamecontrollerdb.txt"
-        "${OpenMW_BINARY_DIR}/Release/openmw.exe"
         DESTINATION ".")
+	INSTALL(FILES "${OpenMW_BINARY_DIR}/Debug/openmw.exe" DESTINATION "." CONFIGURATIONS Debug)
+	INSTALL(FILES "${OpenMW_BINARY_DIR}/Release/openmw.exe" DESTINATION "." CONFIGURATIONS Release;RelWithDebInfo;MinSizeRel)
```

Comment:
`CMAKE_CURRENT_BINARY_DIR` is set to the same as `OpenMW_BINARY_DIR`, so if you really want to avoid the kind of thing that I did, I think it's best to `INSTALL TARGETS` rather than `INSTALL FILES`.  Will get to work on that, let me know if you want to do something different